driver-async/src/test/functional/com/mongodb/async/client/TransactionsTest.java

```diff
@@ -285,18 +285,18 @@ public void execute() {
 
                         assertEquals("Expected operation result differs from actual", expectedResult, actualResult);
                     }
-                    assertFalse(String.format("Expected error '%s' but none thrown", getErrorContainsField(expectedResult)),
-                            hasErrorContainsField(expectedResult));
-                    assertFalse(String.format("Expected error code '%s' but none thrown", getErrorCodeNameField(expectedResult)),
-                            hasErrorCodeNameField(expectedResult));
+                    assertFalse(String.format("Expected error '%s' but none thrown for operation %s",
```

Comment:
Adding operation name to  all the failure messages

Reply:
:+1: